### PR TITLE
Nav bar on map plans and scenarios

### PR DESCRIPTION
**Pull request by pflopez**
Adds a secondary nav bar on map, plans and scenarios.
Screen Shot 2023-08-31 at 09 47 56

Screen Shot 2023-08-31 at 09 48 21

Screen Shot 2023-09-06 at 17 17 43

### Notes: 
- The colors of the ui elements do not match (yet) the figma files, will follow up on this separately 
- The copy link and print buttons do nothing (for now)
- We don't have scenario name for now, so using scenario ID as part of the breadcrumbs.
- The plan name is async and flashes for a second on the nav bar. We might need to rework how we load plans on plan page if we want to avoid this.

### CODE REVIEW

File: src/interface/src/app/plan/plan.component.ts
Comment (sachiyuan) [inline src/interface/src/app/plan/plan.component.ts:135]: More of a clarifying question than a comment: Right now if you start on the map => create a planning area => get to the scenario config page => Click on the back arrow, you get sent home rather than back to the map. Is that the desired flow?

Comment (pflopez) [reply src/interface/src/app/plan/plan.component.ts:135]: if you are in a scenario page you go back to the plan page. If you are on the plan page you go back home

Comment (sachiyuan) [reply src/interface/src/app/plan/plan.component.ts:135]: Ah, sorry mixed up my terms. So even if you're coming from the map, if you're on the plan page then the back arrow is essentially another home button?

Comment (pflopez) [reply src/interface/src/app/plan/plan.component.ts:135]: yes, i can double check if that is the desired behavior on that flow

File: src/interface/src/app/plan/plan.component.ts
Comment (sachiyuan) [inline src/interface/src/app/plan/plan.component.ts:42]: Would you mind actually waiting until my scenario name pr is merged and using that? Realized my changes that are coming soon are going to break this since we won't have a scenario id (configid) until after the scenario is created with the new flow

Comment (pflopez) [reply src/interface/src/app/plan/plan.component.ts:42]: @sachiyuan let me know if this is good to go! thanks :D

Comment (sachiyuan) [reply src/interface/src/app/plan/plan.component.ts:42]: Yes, the scenario name changes are merged!

**Review by sachiyuan — APPROVED** [review]
LGTM

**Pull request merged**
